addons/website_sale_delivery_giftcard/static/tests/tours/website_sale_delivery_gift_card.js

```diff
@@ -1,65 +1,64 @@
-odoo.define('website_sale_delivery_giftcard.test', function (require) {
-    'use strict';
+/** @odoo-module **/
```

Comment:
Since we are in stable, we need to keep the same module name to ensure that overrides are not broken (though highly unlikely) 
```suggestion
/** @odoo-module alias=website_sale_delivery_giftcard.test **/
```